component/testrunner/testrunner.json

```diff
@@ -161,7 +161,7 @@
         },
         "uris" :
         {
-          "add-nocache-param" : true
+          "add-nocache-param" : false
```

Comment:
This feels wrong. I already have difficulty sometimes getting changes to tests to refresh. Without the nocache parameter, things could only get worse, no?